pages/linux/lvcreate.md

```diff
@@ -0,0 +1,19 @@
+# lvcreate
+
+> Creates a logical volume in an existing volume group.
```

Comment:
I think if I explain what is a volume group I need to explain first what is a logical volume and then it will not be a simplified man page. 
Anyone who wants to use it must know what is a volume group due this is an advanced linux command mainly used by sysadmins or people who know what are doing.